### PR TITLE
[Docs] Remaining doc improvements for 3.2.3

### LINKED ISSUE #3386: Our docs should have a document explaining how to update Red

### LINKED ISSUE #3387: [Docs] Shared API key Reference is missing a few methods
Methods are properly all listed as part of the bot base at least right now.

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

- Adds a brief note that the install command doubles as an update command (works toward #3386)
- Documents the Shared API Keys a bit better (Resolves #3387)

### CODE REVIEW

**Comment by Jackenmen** [comment]
I wouldn't call this full resolution of #3386 but rather a good start to adding full update docs at later date.

**Comment by mikeshardmind** [comment]
updated comment so it won't close that issue then.

File: docs/install_linux_mac.rst [hunk 1]
```diff
@@ -355,6 +355,10 @@ Or, to install with PostgreSQL support:
     python -m pip install -U Red-DiscordBot[postgres]
 
 
+.. note::
+
+    This command is also used for updating Red
```
Comment (Jackenmen) [inline docs/install_linux_mac.rst]: We have 2 commands here, this doesn't say which one is supposed to update Red.

File: docs/install_windows.rst [hunk 2]
```diff
@@ -96,6 +96,10 @@ Installing Red
         python -m pip install -U Red-DiscordBot[postgres]
 
 
+.. note::
+
+    This command is also used for updating Red
```
Comment (Jackenmen) [inline docs/install_windows.rst]: We have 2 commands here, this doesn't say which one is supposed to update Red.

**Review by Jackenmen — CHANGES_REQUESTED** [review]

**Review by Jackenmen — APPROVED** [review]
LGTM, looks fine on local build

**Pull request merged**